### ISSUE TITLE
Use StringBuilder instead of String on code generation

### ISSUE THREAD

**Issue by gerazo** [ELTE-Soft/txtUML#233]

**Comment by kovacsgabor** [comment]
What is the status of this issue?

**Comment by gerazo** [comment]
I think we can drop this issue. The reason:
We have decided that there is too much code which is written in this style in the C++ export and would be too much work to change (especially function signatures). On the other hand, when I touched the code in C++ export I did change Strings to StringBuilders at a couple of places where the usage was really frequent (I didn't profile but those places I did change where we were spinning in loops or where adding string parts to the stem had happened multiple times). These changes got into the 1st merge of C++ merges when we merged @nagyandras95 's work back into mainline (I can look it up if it is important). The JVM (or the compiler?) can do optimizations which will use StringBuilders in the back when the usage of a String is obviously dropping the previous instances of the String. While we have no exact knowledge about how this is done, we suppose that it is probably clever enough to cope with the easiest situations: those which were left in the code.
